### ISSUE TITLE
Adding Popularity Slider

### ISSUE THREAD

**Issue by cyrillkuettel** [brianch/offline-chess-puzzles#23]
Hi brianch

First of all thank you for this great project and your efforts at developing it.
The csv contains a field `popularity`. Have you considered adding a slider for this?

**Comment by brianch** [comment]
Hello! And thanks :)

My first thought was to have a low minimum value (just to exclude things that were mostly down-voted in lichess) in the config file (so it's changeable, but not exposed in the UI), since it's not something I imagine that people would tweak much. But yeah, it could also be added as a slider.

I guess I should take a look at the distribution of values to have an idea how it looks, so there's more info to make a decision.

**Comment by cyrillkuettel** [comment]
I was just thinking of setting `popularity=0.99` for example to get only the best of the best;) But yeah config files makes sense I think as this is probably a niche-request

**Comment by brianch** [comment]
Ok, might as well add it as a slider indeed, there's a considerable amount of variation, mostly around the 20 to 100 popularity. (Popularity goes from -100 to 100)

The most popular ones are also the most played, perhaps because lichess recommends them more.

Disclaimer: I don't understand anything about statistics 😅 

![plot](https://github.com/brianch/offline-chess-puzzles/assets/5335499/5118ffd6-3cfb-476f-b7eb-f1b3885c6a48)

![plot-nbplays](https://github.com/brianch/offline-chess-puzzles/assets/5335499/08d776c9-d58f-4335-9200-0bcedbc6197e)

By the way, with this I also found out that apparently there's a few puzzles with more than 3000 rating now, last time I checked there were none, I will adjust the slider in the next release.
